reppy/robots.pyx

```diff
@@ -77,6 +77,7 @@ def FetchMethod(cls, url, ttl_policy=None, max_size=1048576, *args, **kwargs):
         kwargs['stream'] = True
         with closing(requests.get(url, *args, **kwargs)) as res:
             content = res.raw.read(amt=max_size, decode_content=True)
+            logger.debug('Content: %s', content)
```

Comment:
I would probably move this *after* the ContentTooLong exception so that we only see the successful data.